TableauPrep/TableauPrep.munki.recipe

```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Description</key>
+	<string>Downloads latest Tableau Prep disk image and imports into Munki.</string>
+	<key>Identifier</key>
+	<string>ch.unibas.its.git.mcs.munki.tableau-prep</string>
+	<key>Input</key>
+	<dict>
+        <key>NAME</key>
+		<string>TableauPrep</string>
+		<key>MUNKI_REPO_SUBDIR</key>
+		<string>apps/%NAME%</string>
+		<key>pkginfo</key>
+		<dict>
+			<key>blocking_applications</key>
+			<array>
+				<string>%NAME%</string>
+			</array>
+			<key>catalogs</key>
+			<array>
+				<string>testing</string>
+			</array>
+			<key>description</key>
+			<string>Tableau Prep is a self-service data preparation tool with Tableau Prep Builder and Prep Conductor.</string>
+			<key>display_name</key>
+			<string>Tableau Prep</string>
+			<key>installer_choices_xml</key>
```

Comment:
Is this section required? As far as I can tell, it uses the default choices. If that's the case I would like to omit this section. I might be wrong though.

Reply:
I did not remove this section even though it may be the default settings. If we want to further adapt these values during test time, less research is necessary to find these settings. So, is this overhead acceptable or shall I remove it?

Reply:
I would like to omit it if it is not necessary. However, I agree with you that the settings should be documented somewhere.

Could you maybe move them to a README or to the description section?

Reply:
I removed the default settings and kept the ones which are not default. For more information a readme with a link to the possible installer choices is available.

Reply:
I think I wasn't making myself clear. I was meaning to ask whether or not the code provided in the `installer_choices_xml` section does change the behavior of the default Tableau Prep installer. If this is indeed the case (which it appears to be), the code can stay in there.

Can you explain where the choices differ from the defaults and why these choices were made?